geoportal/c2cgeoportal_geoportal/lib/check_collector.py

```diff
@@ -56,6 +56,7 @@ def check(request):
                 )
                 r = requests.get(params={"max_level": str(host.get("max_level", max_level))}, **url_headers)
                 r.raise_for_status()
+                return r.json()
```

Comment:
What's for?

Reply:
Include the result of the health check in the results of this one